h2o-algos/src/main/java/hex/StackedEnsembleModel.java

```diff
@@ -74,10 +76,40 @@ public StackedEnsembleModel(Key selfKey, StackedEnsembleParameters parms, Stacke
       deeplearning
     }
     public MetalearnerAlgorithm _metalearner_algorithm = MetalearnerAlgorithm.AUTO;
-    public String _metalearner_params = new String();
+    public String _metalearner_params = new String(); //used for clients code-gen only.
     public Model.Parameters _metalearner_parameters;
     public long _seed;
 
+    /**
+     * initialize {@link #_metalearner_parameters} with default parameters for the current {@link #_metalearner_algorithm}.
+     */
+    public void initMetalearnerParams() {
+      initMetalearnerParams(_metalearner_algorithm);
+    }
+
+    /**
+     * initialize {@link #_metalearner_parameters} with default parameters for the given algorithm
+     * @param algo the metalearner algorithm we want to use and for which parameters are initialized.
+     */
+    public void initMetalearnerParams(MetalearnerAlgorithm algo) {
+      _metalearner_algorithm = algo;
+      switch (algo) {
+        case AUTO:
+        case glm:
+          _metalearner_parameters = new GLMModel.GLMParameters();
+          break;
+        case gbm:
+          _metalearner_parameters = new GBMModel.GBMParameters();
+          break;
+        case drf:
+          _metalearner_parameters = new DRFModel.DRFParameters();
+          break;
+        case deeplearning:
+          _metalearner_parameters = new DeepLearningModel.DeepLearningParameters();
+          break;
+      }
+    }
+
```

Comment:
I"m thinking about moving this logic somewhere else: don't really like the idea of SEModel having dependencies on all other models. wdyt?

Reply:
tried to move this to Metalearner, along with all those model specific logic for better encapsulation, but this requires bigger refactoring that we probably don't want to include in this PR

Reply:
I think omitting the addition of this method to SEModel is fine. Can add better encapsulation in another PR. How does that sound?

Is the main use of this in AutoML?

Reply:
yes, it's mainly used in AutoML for now, but I have some idea to encapsulate all the metalearner logic in its dedicated class: will do that later and propose a PR.
will leave it as it is now if no one complains